lib/core/include/irods_server_properties.hpp

```diff
@@ -96,6 +101,10 @@ namespace irods
         configuration_parser config_props_;
     }; // class server_properties
 
+    inline bool server_property_exists(const std::string_view _prop) {
```

Comment:
Thoughts on the name `server_property_contains`?

Reply:
I was just thinking about our other key-value things lining up with maps terminology. I am fine with this name too

Reply:
i think `exists` works better given the preceding characters. if we use `contains`, i feel `server_property_` needs to be tweaked, else it sounds like `server_property_contains` is asking `does the server property contain _prop`. hopefully that made sense.

Reply:
Totally